src/control/server/ctl_storage.go

```diff
@@ -155,6 +155,15 @@ func (cs *ControlService) getScmUsage(ssr *storage.ScmScanResponse) (*storage.Sc
 			nss[idx] = ns
 		}
 
+		if nss[idx].Mount != nil {
```

Comment:
This is a bit of a hack but as discussed I can't think of a better way to do it at the moment.